karabo_data/reader.py

```diff
@@ -662,6 +662,51 @@ def get_array(self, source, key, extra_dims=None, roi=by_index[...]):
             sorted(non_empty, key=lambda a: a.coords['trainId'][0]), dim='trainId'
         )
 
+    def get_dask_array(self, source, key):
```

Comment:
What about making this the default method for get_array ?

Reply:
Yes that's exactly what I had in mind. You could make  'xarray' the default and then print a future warning, if 'xarray' was set, that dask will become default. Just an idea though

Reply:
I think I'd leave getting a real array as the default - it's more straightforward. But maybe it could have a warning to suggest using dask if you load more than X amount of data, where X is probably some 10s of GB.